src/main.rs

```diff
@@ -653,3 +668,20 @@ fn remove_arguments_test() {
 
     assert!(arguments.len() == 0);
 }
+
+/// Lists all connected devices
+fn list_connected_devices() -> Result<(), DebugProbeError> {
+    let links = Probe::list_all();
```

Comment:
Why is this called links?

Reply:
Somebody named it like this in the probe-rs-cli ;)